### ISSUE TITLE
Visualize sentiment classifier on gender dataset

### ISSUE THREAD

**Issue by venondev** [braindatalab/gecobench#16]
Classification Analysis:

Is there a difference in sentiment classification when running it on the same sentence with male or female subject?
- Check difference in output e.g. max score 
- Check difference of output probabilities / logits

Does the attribution of tokens change for male or female subject and for the different XAI methods?
